qiskit/transpiler/passmanager.py

```diff
@@ -33,49 +38,102 @@
 class PassManager(BasePassManager):
     """Manager for a set of Passes and their scheduling during transpilation."""
 
-    PASS_RUNNER = RunningPassManager
-
     def __init__(
         self,
-        passes: PassSequence | None = None,
+        passes: Task | list[Task] = None,
```

Comment:
If the intention here is to be `passes: Task | Iterable[Task]`, then a safe default is `()`, which is immutable and doesn't need special handling.